eventsdiscovery/src/main/java/com/schedjoules/client/eventsdiscovery/EventsDiscovery.java

```diff
@@ -84,4 +86,9 @@
      * @return
      */
     EventsDiscovery withStartAtOrAfter(DateTime startAtOrAfter);
+
+    /**
+     * Sets the event categories filter
+     */
+    EventsDiscovery withCategories(Iterable<Uri> categories);
```

Comment:
We have a `Category` model. Wouldn't it make more sense to pass this one instead of the plain `Uri`?